scripts/build.sh

```diff
@@ -86,7 +86,7 @@ curl -sSL https://github.com/contiv/netplugin/releases/download/$contiv_version/
 pushd $output_dir
 tar oxf netplugin-$contiv_version.tar.bz2 netctl
 rm -f netplugin-$contiv_version.tar.bz2
-git clone http://github.com/contiv/ansible
+git clone http://github.com/neelimamukiri/ansible -b netmaster_args
```

Comment:
Looks like this was a local test of the changes on the ansible repo... this can be removed